procedural/constraints/src/main/kotlin/gov/nasa/jpl/aerie/constraints/ActivityId.kt

```diff
@@ -0,0 +1,7 @@
+package gov.nasa.jpl.aerie.constraints
```

Comment:
Do we want this to be `gov.nasa.jpl.aerie.procedural.constraints` (likewise for the timeline library)? Having the timeline library just be `jpl.aerie.timeline` is perfectly clear, but I can see it being a bit confusing having the same namespace as the original constraints library (`:constraints`).

Either way is fine with me, you can resolve this comment if you decide to stick with just `gov.nasa.jpl.aerie.constraints` and `gov.nasa.jpl.aerie.timeline`

Reply:
made this change for the constraints package but not timeline package, like we talked about with Dan (i think)